articles/python-projects-for-beginners/index.md

```diff
@@ -0,0 +1,196 @@
+
+To begin, create a Python file and add:
+
+```python
+import random
+
+user = input("Enter name: ")
+print(f"Welcome {user} to Guessing game101, have fun!")
+
+keep_playing = True
+while keep_playing:
+      print("\nThere are diferent levels: easy, medium and hard")
+      print("\nFor easy, you have 6 chances to guess a number between 1-10. \nFor meduim, you have 4 chances to guess a number between 1-20. \nFor hard, you have 4 chances to guess a number between 1-50")
+      break
+game_level = input("Select desired level: ").lower()
+```
+When you run this, the result in your terminal will be:
+
+![Input field](Input2.JPG)
+
+***While loop*** is used to run a set of commands multiple times as long as the condition is true, i.e., for a while loop to be executed, the condition must be set to '*True*.' The while loop also runs continuously. Therefore we use the ***break*** statement to stop the loop from running.
+
+The user can choose a different desired level. With the ***lower()*** method, the user input will automatically return a lower case string so the program can run without error.
+
+Next, creating the different stages with conditional statements.
+
+```python
+if game_level == "easy":
+  secret_number = random.randint(1,10)
+  guess_count = 0
+  guess_limit = 6
+  while guess_count < guess_limit:
+        guess = int(input("Guess:  "))
+        guess_limit -= 1
+        if guess == secret_number:
+            print("You got it right!")
+            break
+        else:
+          print("That was wrong")
+          print(f"You have {guess_limit} guesses left")
+
+  else:
+      print("Game Over!")
+
+elif game_level == "medium":
+    secret_number = random.randint(1,20)
+    guess_count = 0
+    guess_limit = 4
+    while guess_count < guess_limit:
+       guess = int(input("Guess:  "))
+       guess_count += 0
+       guess_limit -= 1
+       if guess == secret_number:
+          print("You got it right!")
+          break
+       else:
+          print("That was wrong")
+          print(f"You have {guess_limit} guesses left")
+    else:
+      print("Game Over!")
+
+elif game_level == "hard":
+    secret_number = random.randint(1,50)
+    guess_count = 0
+    guess_limit = 4
+    while guess_count < guess_limit:
+        guess = int(input("Guess:  "))
+        guess_limit -= 1
+        if guess == secret_number:
+            print("You got it right!")
+            break
+        else:
+          print("That was wrong")
+          print(f"You have {guess_limit} guesses left")
+    else:
+      print("Game Over!")
+
+else:
+    print("Invalid, try easy, medium or hard")
+```
+
+Let's see this in our terminal.
+
+![guessing game](Guessinggame.png)
+
+The ***randint*** method returns an integer from a random range. Notice the indentations? In Python, indentation is important, and improper indentation will cause an error, or the codes may not run as desired.

```

Comment:
After the second project, add a repl link where the reader can directly go and run the code. 
For example: 

You can run the code [here](https://repl.it/@lalithNarayan/SoreMediumblueMemory#main.cs)